src/QuickMarcEditor/QuickMarcEditorRows/BytesField/BytesField.js

```diff
@@ -217,6 +218,18 @@ const renderSubField = (name, config) => {
   }
 
   const getMaxLengthByType = config.type === SUBFIELD_TYPES.BYTE ? FIXED_FIELD_MAX_LENGTH : config.length;
+  const className = classNames(
+    styles[`bytesFieldSubField${config.type}`],
+    {
+      [styles.noLabelField]: config.noLabel,
+      [styles.disabledField]: config.disabled,
+      [styles.width40]: config.width40,
```

Comment:
We also can extend BE config.

Reply:
I doubt that back-end team will agree to add such styling properties to the config because it's presentation specific and shouldn't be stored on back-end side